translib/transformer/xlate_to_db.go

```diff
@@ -19,20 +19,21 @@
 package transformer
 
 import (
-    "fmt"
-    "github.com/openconfig/ygot/ygot"
-    "os"
-    "reflect"
-    "strings"
-    "github.com/Azure/sonic-mgmt-common/translib/db"
-    "github.com/Azure/sonic-mgmt-common/translib/ocbinds"
-    "github.com/Azure/sonic-mgmt-common/translib/tlerr"
-    "github.com/openconfig/ygot/ytypes"
-    "github.com/openconfig/goyang/pkg/yang"
-    log "github.com/golang/glog"
+	"fmt"
+	"os"
+	"reflect"
+	"strings"
+
+	"github.com/Azure/sonic-mgmt-common/translib/db"
+	"github.com/Azure/sonic-mgmt-common/translib/ocbinds"
+	"github.com/Azure/sonic-mgmt-common/translib/tlerr"
+	log "github.com/golang/glog"
+	"github.com/openconfig/goyang/pkg/yang"
+	"github.com/openconfig/ygot/ygot"
+	"github.com/openconfig/ygot/ytypes"
 )
 
-var ocbSch, _ = ocbinds.Schema()
+var ocbSch, _ = ocbinds.GetSchema()
```

Comment:
Can you keep it as ` ocbinds.Schema()` itself so that compilation goes through? Then you will not have have tight dependency on the other PR that introduces `ocbinds.GetSchema()` API. I will take care of changing this reference also as part of that PR (similar to https://github.com/project-arlo/sonic-mgmt-common/commit/3863507f)